docs/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnType": false
+}
```

Comment:
We aren’t typically including vscode settings file. Was this included by mistake?

Reply:
yep, will remove!